zingolib/src/lightclient/send.rs

```diff
@@ -53,6 +60,23 @@ impl LightClient {
             .collect()
     }
 
+    /// Unstable function to expose the zip317 interface for development
```

Comment:
doc-test, or unit-test, or TODO

Reply:
i can add a TODO but this function is clearly still under a lot of development anyways

Reply:
OK.  This fulfills my requests.